app/templates/index.hbs

```diff
@@ -1,12 +1,22 @@
 <div class="greeting">
   <h1>Welcome to my site!</h1>
 </div>
-<UserStatus 
-  @status={{this.status}} 
-  @changeStatus={{this.changeStatus}} 
-  @isStatusUpdating={{this.isStatusUpdating}}
-  @updateStatus={{this.updateStatus}}
-/>
+{{#if this.isDevMode}}
+  <UserStatus 
+    @status={{this.status}} 
+    @changeStatus={{this.changeStatus}}
+    @dev={{this.isDevMode}} 
+    @isStatusUpdating={{this.isStatusUpdating}}
+    @updateStatus={{this.updateStatus}}
+  />
+  {{else}}
+  <UserStatus 
+    @status={{this.status}} 
+    @changeStatus={{this.changeStatus}} 
+    @isStatusUpdating={{this.isStatusUpdating}}
+    @updateStatus={{this.updateStatus}}
+  />
+{{/if}}
```

Comment:
Does <UserStatus /> required to write two times
if @dev=false pass then cancel ooo button not render
if @dev=true pass then cancel ooo button render

Reply:
```suggestion
{{#if this.isDevMode}}
  <UserStatus 
    @status={{this.status}} 
    @changeStatus={{this.changeStatus}}
    @dev={{this.isDevMode}} 
    @isStatusUpdating={{this.isStatusUpdating}}
    @updateStatus={{this.updateStatus}}
  />
```